src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -939,6 +928,47 @@ def body(self):
         finally:
             os.remove(config_file)
 
+class VMSSExtensionInstallTest(VCRTestBase):
```

Comment:
That works for me. I will probably create a simple PR that we make anything used VCRTestBase fail if it is not in playback to ensure that it is updated if the test ever needs to be re-recorded.

---
In reply to: [76143673](https://github.com/Azure/azure-cli/pull/737#discussion_r76143673) [](ancestors = 76143673,76134986,76100732,76078126)